apollo-ios/Sources/Apollo/Constants.swift

```diff
@@ -1,5 +1,6 @@
 import Foundation
 
 public enum Constants {
-    public static let ApolloVersion: String = "1.18.0"
+  public static let ApolloName = "apollo-ios"
```

Comment:
Can we add a comment of how this is being used. And maybe change it to `ApolloClientName`

Reply:
I'll rename the version constant too; they should both reflect their _client_ intent.